### DISCUSSION TITLE
增加一个倒计日的功能

### DISCUSSION THREAD

**Discussion by jason14250** [Class-Widgets/Class-Widgets#166] [建议]
能不能添加一个倒计日的功能？

**Comment by RinLit-233-shiroko** [comment]
啊……
![image](https://github.com/user-attachments/assets/c82b6f09-ccc1-479b-a57c-a682e8daf46a)
你说的是这个吗

**Reply by jason14250** [reply]
不好意思，这个我没发现，不过还有就是倒计日能不能做一个效果，起强调作用，比如打开后把字体改成醒目一点的并且变成红色，大概的效果可以看一下另一个桌面课程表项目，用electron写的，也在github上面https://github.com/EnderWolf006/ElectronClassSchedule，具体效果如图  ![mmexport1733579648507.png](https://github.com/user-attachments/assets/b195a447-0a8f-4333-811e-fe5d5e199183)

**Reply by RinLit-233-shiroko** [reply]
> 不好意思，这个我没发现，不过还有就是倒计日能不能做一个效果，起强调作用，比如打开后把字体改成醒目一点的并且变成红色，大概的效果可以看一下另一个桌面课程表项目，用electron写的，也在github上面[https://github.com/EnderWolf006/ElectronClassSchedule，具体效果如图](https://github.com/EnderWolf006/ElectronClassSchedule%EF%BC%8C%E5%85%B7%E4%BD%93%E6%95%88%E6%9E%9C%E5%A6%82%E5%9B%BE) ![mmexport1733579648507.png](https://private-user-images.githubusercontent.com/183632427/393521309-b195a447-0a8f-4333-811e-fe5d5e199183.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM1ODAxMjAsIm5iZiI6MTczMzU3OTgyMCwicGF0aCI6Ii8xODM2MzI0MjcvMzkzNTIxMzA5LWIxOTVhNDQ3LTBhOGYtNDMzMy04MTFlLWZlNWQ1ZTE5OTE4My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwN1QxMzU3MDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03NTRmMzA2MzE5YWY4NzNjMWE4N2Y4NTlkNzYxMzczM2IxMGY3OTcxMWYyMTdkZGFhMWE1Y2U3ZTUwZTg1NmVmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Ere6yU3D-pTJ-yk_irnRJElaeTllM6PD98k_XpMSytk)

好，会采纳的（您也可以尝试使用Qt Designer对ui文件进行更改）

**Reply by jason14250** [reply]
就是说给一个选项，打开后就是这个效果，不打开就是目前默认的效果

**Reply by RinLit-233-shiroko** [reply]
> 就是说给一个选项，打开后就是这个效果，不打开就是目前默认的效果

了解，会给用户自定义选项的
